clio-util/src/main/scala/org/broadinstitute/clio/util/generic/CaseClassMapper.scala

```diff
@@ -43,7 +43,7 @@ class CaseClassMapper[T: ClassTag] {
   /** Creates an instance of T from a Map of field names to values. */
   def newInstance(vals: Map[String, _]): T = {
 
-    /** When there is no value for an field, default the value as None, or throw an error if the type isn't Option. */
+    /* When there is no value for a field, default the value as None, or throw an error if the type isn't Option. */
```

Comment:
Why is this not javadoc/scaladoc?

Reply:
Scaladoc (apparently) doesn't bind to nested methods, so this was causing errors to print during `sbt publish`.

Reply:
whaat

Reply:
It's spelled "wat".